### PR TITLE
Add post on intro to Git and Github -- Sarah Gao

### PR DESCRIPTION
**Pull request by sarah-gao**
Here's my blog post introducing beginners to the ideas of Git and Github as well as some linked resources for best practices.

Ready for review, thanks!

### CODE REVIEW

File: _posts/2020-11-06-intro-to-git-github.md [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+* Use active voice
+  * E.g. "Fix spelling errors" is better than "Spelling errors fixed"
+* Use the imperative mood
+  * This is the same tense as when you're giving a command
+  * Follows Git's default commit style
+  * If written correctly, it should seamlessly fit into the template: "If applied, your commit message will *your commit message*"
+  * E.g. "Add README.md" is better than "Added README.md"
+* Keep the subject line to <50 characters
+* Capitalize the subject online
+* Wrap the body text, if used, at 72 characters for easier legibility
```
Comment (Quentin-Smith) [inline _posts/2020-11-06-intro-to-git-github.md:57]: You formatted this very nicely! It gets exactly everything you need.

File: _posts/2020-11-06-intro-to-git-github.md [hunk 2]
```diff
@@ -0,0 +1,93 @@
...
+
+#### Using pull requests
+
+You can also utilize pull requests on Github when you're collaborating with others, or even within your own personal projects. A pull request tells others that you want to merge these changes you've made to the base branch. This allows the code to stay neatly compartmentalized while team members discuss the proposed code and changes within the pull request. Once it's approved, those changes can then be merged into the base branch. To learn more about pull requests, check out [Github's documentation here](https://docs.github.com/en/free-pro-team@latest/github/collaborating-with-issues-and-pull-requests/about-pull-requests)
+
+***
+
+And that's all for now! Hopefully this gives you a glimpse into the world of Git and Github -- two extremely useful and quintessential tools any beginner programmer should start using as soon as possible. They can save a lot of heartache and wasted energy and it's better to start off with best practices as soon as possible.
+
+Happy coding!
```
Comment (Quentin-Smith) [inline _posts/2020-11-06-intro-to-git-github.md:93]: I enjoyed the overall tone and writing of this post. Great job!

**Review by Quentin-Smith — COMMENTED** [review]
One thing Dr. Z mentioned was trying to limit how many "be" verbs there are in our posts. These verbs are like: is, be, was, were, etc. It helps the flow if you remove these.

For instance the second sentence is "This post is intended to serve as a brief beginner's introduction to the concept of Git and a sampling of some of the most commonly used workflows for this nifty tool." That could be rewritten as "This post *intends* to serve as a brief beginner's introduction to the concept of Git and a sampling of some of the most commonly used workflows for this nifty tool."

Sometimes it's that simple but other times it requires you to rewrite entire paragraphs since they hinge on those verbs. 

You put in a few contractions, "You'll, you're, it's," all these contain those verbs as well. Completely changing your writing style in one sitting is near impossible but it helps to keep it in mind.

You do not need to get rid of all of them when writing but they start to weigh things down they pile up. 

Also, I'm not sure if this is more of a quirk of my personal writings but I prefer to limit the amount of adverbs (words like basically -- end in -ly). Like the "be verbs" they bloat writing.

Beyond those things great work!

**Review by mhaliz — APPROVED** [review]
This is a thorough and well done intro to Git and Github! You cover all the main points clearly and concisely, and point to links if the reader wants to learn more. I'm going to merge this onto the class website.

**Pull request merged**